### ISSUE TITLE
How can i annotate my eukaryotic MAGs by metaeuk?

### ISSUE THREAD

**Issue by YoungerRen** [soedinglab/metaeuk#92]
Dear Elileka:
After I extracted the eukaryotic sequences with metarep, I binned the eukaryotic sequences with metawrap, and the resulting MAGs were evaluated for completeness with busco, in which the metaeuk mode was invoked, and the output file contained the metaeuk_output file. Then I want to annotate the MAGs for species, using metaeuk taxtocontig, but I don't quite understand which input files should be? Can you help me with this?

following was the reference code:
metaeuk taxtocontig <i:contigsDB> <i:predsResults.fas> <i:predsResults.headersMap.tsv> <i:taxAnnotTargetDb> <o:taxResult> <tmpDir> --majority 0.5 --tax-lineage 1 --lca-mode 2
these were my metaeuk_output:
![image](https://github.com/user-attachments/assets/acca6dd9-a333-4205-9ef4-3ac4b35c54df)

So do u explain the input file ? or  what files do i prepare for metaeuk taxtocontig?

best 
Yonger

**Comment by YoungerRen** [comment]
My main question is that :what's this data <i:taxAnnotTargetDb>,  it's not mentioned in your readme file, only seqTaxDb, are these two the same file?

**Comment by elileka** [comment]
Yes, I am sorry if that was confusing, these two are the same :)

**Comment by elileka** [comment]
The files you need are listed here.
The following files you should obtain through your BUSCO run. I haven't run BUSCO myself so I am not sure where they are found inside the directories BUSCO produces... 
<i:contigsDB> <i:predsResults.fas> <i:predsResults.headersMap.tsv> 

This file is indeed the seqTaxDb, which can be dowloaded using the `databases` command, for example:
<i:taxAnnotTargetDb> 

This is the prefix for the output that will be generated by the call:
<o:taxResult>
